xchange-openexchangerates/pom.xml

```diff
@@ -29,6 +29,18 @@
             <version>${project.version}</version>
         </dependency>
 
+        <!-- PowerMockito for unit tests -->
+        <dependency>
+            <groupId>org.powermock</groupId>
+            <artifactId>powermock-api-mockito2</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>org.powermock</groupId>
+            <artifactId>powermock-reflect</artifactId>
```

Comment:
Please define this in the parent POM and use the same version as the other powermock dependency.

Reply:
Fixed, i upgraded powermock to 2.0.0-RC.1 ( was 2.0.0-beta.5) in order to make it work using the same version.